x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/signals/open_close_signals_route.ts

```diff
@@ -36,7 +36,7 @@ export const setSignalsStatusRouteDef = (server: ServerFacade): Hapi.ServerRoute
         queryObject = query;
       }
       try {
-        return callWithRequest(request, 'updateByQuery', {
```

Comment:
This line made me realize that we should really rename this to `const callCluster = callClusterFactory(request, server)` for consistency.